nus-cs2103-AY2425S1/ip
# [Chiang Qin Kang] iP

# NotAGPT

[**NotAGPT**](https://github.com/chiangqinkang/ip)

NotAGPT helps store tasks, to remind you when they're due. It's

- Test based
- _Easy to use_
- **Lightweight** 😉
All you need to do is

1. Make sure you have Java 17 installed
2. Run the JAR

- If not, download it from the Java website
- If it does not work, make sure your version is Java 17.
- You can do this with `java -version` in your command line.

Features

- [ ] Stores tasks
- [ ] Stores deadlines
- [ ] Saves tasks

> [!NOTE]
> This is a CLI application.

```java
public class Main {
    public static void main(String[] args) {
        Application.launch(MainApp.class, args);
    }
}```
